app/views/lookbook/previews/show.html.erb

```diff
@@ -17,8 +17,24 @@
         <% end %>
       <% end %>
 
-      <% toolbar.section divide: :left, align: :right, class: "flex-none relative z-10" do %>
+      <% toolbar.section align: :right, class: "flex-none" do %>
+        <% if @dynamic_display_options.any? %>
+          <%= lookbook_render "display_options/editor" do |editor| %>
+            <% @dynamic_display_options.each do |key, opts| %>
+              <% editor.field name: key, opts: opts, value: @static_display_options[key] %>
+            <% end %>
+          <% end %>
+        <% end %>
+      <% end %>
+
+      <% toolbar.section divide: :left, class: "flex-none relative z-10" do %>
         <%= lookbook_render :button_group do |group| %>
+          <% group.button icon: :link,
+            tooltip: "Copy preview URL",
+            copy: true do %>
+            <%= lookbook_inspect_url(@target.lookup_path, request.query_parameters) %>
```

Comment:
I would expect that the copy preview url would include the current state of the `static_display_options`. So that if I clicked on copy, it would include `?_display=theme%3Adark` even if that's not currently in the browser url.

Reply:
@jonrohan urgh yes it should - that was the whole point of it! Just me not paying enough attention before bed. Will fix that, thanks.

Reply:
@jonrohan should be all fixed now 👍